venue-api-mapbox/public/index.html

```diff
@@ -39,10 +39,10 @@
     }
 
     #button-container {
-      size: auto;
-      position: absolute;
-      top: 40%;
-      right: 20px;
+      position: relative;
+      left: 94%;
```

Comment:
Do not relative to "left" if the buttons need to be in the right edge (this is probably the cause of the issues with smaller screen sizes)

Reply:
Its not actually related to the buttons hiding. They hide because button container is whole page wide and the position relative does not work correctly